contrib/newsraft/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "newsraft"
+pkgver = "0.22"
+pkgrel = 0
+build_style = "makefile"
+hostmakedepends = [
+    "scdoc",
+    "pkgconf",
+]
+makedepends = [
+    "gumbo-devel",
+    "libcurl-devel",
+    "libexpat-devel",
+    "ncurses-devel",
+    "sqlite-devel",
+    "yajl-devel",
+]
+pkgdesc = "Feed reader for terminal"
+maintainer = "ttyyls <contact@behri.org>"
+license = "ISC"
+url = "https://codeberg.org/newsraft/newsraft"
+source = f"{url}/archive/{pkgname}-{pkgver}.tar.gz"
+sha256 = "16dbda602a37a2c8052421e90dd00ca065bab4130c6c2c16d5ca2a8a184990b9"
+hardening = ["vis", "cfi"]
+# relies on symbols being exposed in the shared library
+options = ["!check"]
```

Comment:
what shared library? this is one executable

if symbols are missing in something else then it should not have =vis

Reply:
ah, i see. there is a separate libnewsraft.so that is used only for testing

sure, then remove vis

Reply:
besides that, it still errors on parsing various date formats.

Reply:
I think enabling vis and cfi is more valuable than running those tests anyway.